x-pack/plugins/actions/server/plugin.ts

```diff
@@ -345,6 +323,16 @@ export class ActionsPlugin implements Plugin<Promise<PluginSetupContract>, Plugi
     const getScopedSavedObjectsClientWithoutAccessToActions = (request: KibanaRequest) =>
       core.savedObjects.getScopedClient(request);
 
+    const spaceIdToNamespace = (spaceId?: string) => {
+      return plugins.spaces && spaceId
+        ? plugins.spaces.spacesService.spaceIdToNamespace(spaceId)
+        : undefined;
+    };
+
+    const encryptedSavedObjectsClient = plugins.encryptedSavedObjects.getClient({
+      includedHiddenTypes,
+    });
+
```

Comment:
This code used to reside outside of `getActionsClientWithRequest` to prevent us from creating a new `encryptedSavedObjectsClient` every time an ActionsClient is created.
Is there a reason why this has been moved in now? 🤔